src/@types/OceanNode.ts

```diff
@@ -8,6 +8,12 @@ export interface OceanNodeDBConfig {
   url: string | null
 }
 
+// blacklisted peer ids and ips (for rate limiting purposes)
+export interface BlackList {
```

Comment:
We've always used the terms allow list / deny list for all access control. I suggest doing the same here:

```suggestion
export interface DenyList {
```

Reply:
really?  ok :-)